src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerExecContainer.groovy

```diff
@@ -20,38 +20,45 @@ import org.gradle.api.tasks.Optional
 
 class DockerExecContainer extends DockerExistingContainer {
 
-  	@Input
-  	String[] cmd
+    @Input
+    String[] cmd
 
-  	@Input
-  	@Optional
-  	Boolean attachStdout = true
+    @Input
+    @Optional
+    Boolean attachStdout = true
 
-  	@Input
-  	@Optional
-  	Boolean attachStderr = true
+    @Input
+    @Optional
+    Boolean attachStderr = true
 
+    String execId
```

Comment:
Just merged this PR but wondering if we should make this private and provide a method to retrieve it's value? Might give the impression that it can or should be set by developers.